### PR TITLE
Staff Log: April 2019

### PR DESCRIPTION
**Pull request by MegaApplePi**

### CODE REVIEW

File: wiki/People/Staff_Log/2019/en.md [hunk 1]
```diff
@@ -65,3 +65,30 @@ All users listed below were added on 2019-02-16.
...
+- Added [Zeus-](https://osu.ppy.sh/users/5464437) to the Global Moderation Team (2019-04-23)
+- Added [- Felix](https://osu.ppy.sh/users/8503985) to the Global Moderation Team (2019-04-23)
+
+### Beatmap Nominator removals
+
+- [MaridiuS](https://osu.ppy.sh/users/4496961) - resignation (2019-04-01)
+- [alienflybot](https://osu.ppy.sh/users/636114) - resignation (2019-04-12)
+- [Yales](https://osu.ppy.sh/users/2377881) - resignation (2019-04-16)
+- [DMT9 Nowa](https://osu.ppy.sh/users/5428909) - resignation (2019-04-27)
+- [wonjae](https://osu.ppy.sh/users/5032045) - resignation (2019-04-25)
```
Comment (WalterToro) [inline wiki/People/Staff_Log/2019/en.md]: This should be over DMT9 if we are going chronologically

File: wiki/People/Staff_Log/2019/en.md [hunk 2]
```diff
@@ -65,3 +65,30 @@ All users listed below were added on 2019-02-16.
 ## March
 
 - Moved [Mir](https://osu.ppy.sh/users/8688812) to the Quality Assurance Team (2019-03-23)
+
+## April
+
+- Moved [Fycho](https://osu.ppy.sh/users/1876867) to the Global Moderation Team (2019-04-10)
```
Comment (WalterToro) [inline wiki/People/Staff_Log/2019/en.md]: Missing Ascendance being added to the QAT.

`- Added [Ascendance](https://osu.ppy.sh/users/2931883) to the Quality Assurance Team (2019-04-10)`

File: wiki/People/Staff_Log/2019/en.md [hunk 3]
```diff
@@ -65,3 +65,30 @@ All users listed below were added on 2019-02-16.
...
+- Moved [Loctav](https://osu.ppy.sh/users/71366) to the Global Moderation Team (2019-04-22)
+- Added [Zeus-](https://osu.ppy.sh/users/5464437) to the Global Moderation Team (2019-04-23)
+- Added [- Felix](https://osu.ppy.sh/users/8503985) to the Global Moderation Team (2019-04-23)
+
+### Beatmap Nominator removals
+
+- [MaridiuS](https://osu.ppy.sh/users/4496961) - resignation (2019-04-01)
+- [alienflybot](https://osu.ppy.sh/users/636114) - resignation (2019-04-12)
+- [Yales](https://osu.ppy.sh/users/2377881) - resignation (2019-04-16)
+- [DMT9 Nowa](https://osu.ppy.sh/users/5428909) - resignation (2019-04-27)
```
Comment (Joehuu) [inline wiki/People/Staff_Log/2019/en.md]:
```suggestion
- [wonjae](https://osu.ppy.sh/users/5032045) - resignation (2019-04-25)
- [DMT9 Nowa](https://osu.ppy.sh/users/5428909) - resignation (2019-04-27)
```

File: wiki/People/Staff_Log/2019/en.md (hunk 1 above)
Comment (Joehuu) [inline wiki/People/Staff_Log/2019/en.md]:
```suggestion
```

**Comment by Joehuu** [comment]
PR seems to be broken.

**Comment by MegaApplePi** [comment]
Those random commits shouldn't change anything since they were already merged into ppy:master, but fixed it anyways.

**Pull request merged**